src/main/java/com/synopsys/integration/blackduck/imageinspector/containerfilesystem/DataStripper.java

```diff
@@ -0,0 +1,17 @@
+package com.synopsys.integration.blackduck.imageinspector.containerfilesystem;
+
+public class DataStripper {
+    
+    public static String stripEpocFromVersion(String version) {
+        return version != null && version.startsWith("0:") ? version.substring(2) : version;
```

Comment:
Do we need to do a `strip()` in case of a space before `0`. Example `" 0:"`. Would it skip stripping epoch if first char is space?

Reply:
Not required because the String.format("%s-%s") is used to get the version. That implicitly strips blanks before the version.